corehq/apps/receiverwrapper/tests/test_submit_errors.py

```diff
@@ -52,6 +56,9 @@ def testSubmitDuplicate(self):
 
         file, res = self._submit('simple_form.xml')
         self.assertEqual(201, res.status_code)
+
+        _, res_openrosa3 = self._submit('simple_form.xml', open_rosa_header=OPENROSA_VERSION_3)
+        self.assertEqual(422, res_openrosa3.status_code)
```

Comment:
I have first written separate test, but there is bunch of unavoidable duplication, so I added it here.